README.md

```diff
@@ -10,8 +10,8 @@ $ ./bin/vpcanalyzer -h
 Usage of vpc-network-config-analyzer:
   -analysis-type string
         Supported analysis types:
-        * all_endpoints  - supported with: txt, md, json, drawio, arch_drawio, debug
-        * all_subnets  - supported with: txt, md, json, drawio, arch_drawio
+        * all_endpoints  - supported with: txt, md, json, drawio, arch_drawio, svg, arch_svg, html, arch_html, debug
+        * all_subnets  - supported with: txt, md, json, drawio, arch_drawio, svg, arch_svg, html, arch_html
```

Comment:
why is this change related to this PR?

Reply:
not related, it was not added to README by the one who added these lines to parse_args

Reply:
oh. ok